capabilities/loosely-coupled-teams.md

```diff
@@ -1,46 +1,46 @@
 # [Loosely Coupled Teams](https://dora.dev/capabilities/loosely-coupled-teams/)
 
-Teams that can implement large-scale changes independently—without needing to coordinate with other teams—tend to be significantly more effective. This loose coupling is enabled by team structure, charter, and architecture. When teams fully own their subsystems and have reliable integration points within the larger system, they can make changes quickly by reducing the wait times typically associated with cross-team coordination.
+Teams that can implement large-scale changes independently, without the need to coordinate with other teams, are often referred to as loosely coupled teams, and they tend to be significantly effective. Loose coupling is enabled by team structure, charter, and architecture. When teams fully own their subsystems and have reliable integration points within the larger system, they can make changes quickly and reduce the wait times typically associated with cross-team coordination.
```

Comment:
Describing something as significantly effective sounds odd. It use to be teams were significantly more effective. I'm not attached to the original, but this will need an additional tweak.

Reply:
All of the other changes in this line look good to me!